lib/rubocop/cop/style/file_name.rb

```diff
@@ -17,6 +17,14 @@ class FileName < Cop
 
         SNAKE_CASE = /^[\da-z_.?!]+$/
 
+        ACRONYM_NAME_MAPPINGS = {
+          'Cli'  => 'CLI',
+          'Dsl'  => 'DSL',
+          'Http' => 'HTTP',
+          'Rfc'  => 'RFC',
+          'Xml'  => 'XML'
+        }.freeze
```

Comment:
I think this list is incomplete(e.g. `URI` should be included into the list).
Maybe `golint` source code is helpful. https://github.com/golang/lint/blob/master/lint.go#L731-L770